tests/std/tests/P0896R4_ranges_alg_fill/test.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <range_algorithm_support.hpp>
+#include <ranges>
+#include <utility>
+
+constexpr void smoke_test() {
+    using ranges::fill, ranges::iterator_t;
+    using std::same_as;
+
+    int output1[] = {13, 42, 1367};
+    int output2[] = {13, 42, 1367};
+    {
+        const int value = 7;
+        auto result     = fill(std::begin(output1), std::end(output1), value);

```

Comment:
Ewww, `std::begin` and `std::end` in a Ranges test. Not that it makes any difference other than to get some more bits of test coverage for other components, but these could all be `ranges::begin`/`ranges::end`.